manifests/openshift/kepler/01-kepler-install.yaml

```diff
@@ -60,7 +60,8 @@ spec:
         app.kubernetes.io/name: kepler-exporter
     spec:
       nodeSelector:
-        sustainable-computing.io/kepler: ""
+        #remove and add node annotation if running in non-development
+        #sustainable-computing.io/kepler: ""
```

Comment:
If we we are commenting out this nodeSelector it will result in an invalid array so we will need to comment out `spec.nodeSelector` or put something like `kubernetes.io/os: linux`. Probably adding a default nodeSelector for it to be `kubernetes.io/os: linux` will also help with mixed cluster (linux and non-linux nodes) as currently Kepler is known to work only on Linux based OS.

Reply:
I added `kubernetes.io/os: linux` there, thanks!